infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -274,15 +275,15 @@ EOT
       target_host : "api.github.com"
       source_auth_strategy : "oauth2_refresh_token"
       secured_variables : [
-        { name : "ACCESS_TOKEN", writable : true },
```

Comment:
:red_circle: I think this is not going to work. ACCESS_TOKEN needs to exist if using `USE_SHARED_TOKEN : "TRUE"` as it will try to persist it.
If not, will be checking for the token (might fail) or generating a new token every single call I believe.